_posts/2016-1-25-Blog-Building.md

```diff
@@ -0,0 +1,158 @@
+			<a href="http://chrispederick.com/work/web-developer/" target="\_blank">Web Developer Browser Plug-In</a>
+		</td>
+
+		<td>
+			<center><a href="http://chrispederick.com/work/web-developer/" target="\_blank"><i class="fa fa-cog fa-5x" style="color: gray;"></i></a></center>
+		</td>
+
+		<td>
+			This plugin, available on Chrome, Firefox, and Opera, is super useful for testing. I use it to test how the blog will look for people with color-blindness, people who have CSS disabled, and people who may be reading the blog with only screen readers.
+		</td>
+	</tr>
+
+	<tr>
+		<td>
+			<a href="http://fortawesome.github.io/Font-Awesome/" target="\_blank">Font Awesome</a>
+		</td>
+
+		<td>
+			<center><a href="http://fortawesome.github.io/Font-Awesome/" target="\_blank"><img src="/images/Logos/font-awesome_logo.png" alt="Font Awesome Logo"></img></a></center>
+		</td>
+
+		<td>
+			Font Awesome is a free font of logos and icons. I use it on the blog to create the buttons at the bottom of the page to link to an RSS feed and my Github and LinkedIn accounts. I used in the line above to create the gear icon!
+		</td>
+	</tr>
+
+	<tr>
+		<th colspan="3">Techniques</th>
+	</tr>
+
+	<tr>
+		<td>
+			Run the blog locally before posting
+		</td>
+
+		<td>
+			<center><img src="/images/Logos/cmd_logo.png" alt="Command Prompt Logo"></img></center>
+		</td>
+
+		<td>
+			It's very helpful to see what your blog will look like as you're building it and before you show it to everyone else. To do this, I use my Command Prompt (just search of it on your computer) to navigate into my blog directory and then serve it on my local computer. The commands are super basic:
+			<script src="https://gist.github.com/madjenjen/f23aa17199c2fcf3c0ec.js"></script>
+			Then you'll use your web browser to go to <a href="http://127.0.0.1:4000" target=\_blank">http://127.0.0.1:4000</a> to check out your blog!
+
+			Fun fact: Jekyll is smart enough to know the date you want your blog posted, and it won't show it even locally if you put a future date on it, so for testing purposes, title your blog with today's date and then change it when you're ready to push it to the world.
+
+		</td>
+	</tr>
+
+	<tr>
+		<td>
+			Use git for source control and repository
+		</td>
+
+		<td>
+			<center><a href="https://github.com/madjenjen/madjenjen.github.io" target="\_blank"><img src="/images/Logos/GitHub_logo.png" alt="GitHub Logo"></a></img></center>
+		</td>
+
+		<td>
+			This blog is built using the free GitHub Pages feature of GitHub, so it makes sense to use git for source control as well. I downloaded <a href="https://www.cygwin.com/" target="\_blank">Cygwin</a> for my Windows computer (you won't have to do this if you're using a Mac) and use it to save my blog changes locally and then push them to the master repository when they're dece enough for the world to see! Check out this <a href="https://try.github.io/levels/1/challenges/1" target="\_blank">15 minute tutorial on how to use git</a>.
```

Comment:
[Git Bash](https://git-scm.com/downloads) is often used on Windows as well and is easier to install. Might be worth adding a link.